src/library/Uno.Themes.WinUI.Markup/Theme.LightWeight/Themes.LightWeightStyles.CheckBox.cs

```diff
@@ -0,0 +1,245 @@
+							public static ResourceValue<Brush> Indeterminate => new("CheckBoxGlyphForegroundIndeterminate", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminatePointerOver")]
+							public static ResourceValue<Brush> IndeterminatePointerOver => new("CheckBoxGlyphForegroundIndeterminatePointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminatePressed")]
+							public static ResourceValue<Brush> IndeterminatePressed => new("CheckBoxGlyphForegroundIndeterminatePressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminateDisabled")]
+							public static ResourceValue<Brush> IndeterminateDisabled => new("CheckBoxGlyphForegroundIndeterminateDisabled", true);
+						}
+
+						public static class PathData
+						{
+							[ResourceKeyDefinition(typeof(string), "CheckBoxHyphenGlyphPathData")]
+							public static ResourceValue<string> Hyphen => new("CheckBoxHyphenGlyphPathData", true);
+
+							[ResourceKeyDefinition(typeof(string), "CheckBoxCheckGlyphPathData")]
+							public static ResourceValue<string> Check => new("CheckBoxCheckGlyphPathData", true);
+						}
+					}
+
+					public static class Typo
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "CheckBoxFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("CheckBoxFontFamily", true);
+#if WinUI
+						[ResourceKeyDefinition(typeof(FontWeights), "CheckBoxFontWeight")]
+						public static ResourceValue<FontWeights> FontWeight => new("CheckBoxFontWeight", true);
+#else
+						[ResourceKeyDefinition(typeof(FontWeight), "CheckBoxFontWeight")]
+						public static ResourceValue<FontWeight> FontWeight => new("CheckBoxFontWeight", true);
+#endif
+						[ResourceKeyDefinition(typeof(double), "CheckBoxFontSize")]
+						public static ResourceValue<double> FontSize => new("CheckBoxFontSize", true);
+
+						[ResourceKeyDefinition(typeof(double), "CheckBoxCharacterSpacing")]
+						public static ResourceValue<double> CharacterSpacing => new("CheckBoxCharacterSpacing", true);
+					}
+
+					public static class FocusArea
+					{
+						public static class Size
+						{
+							[ResourceKeyDefinition(typeof(double), "CheckBoxFocusAreaSize")]
+							public static ResourceValue<double> Default => new("CheckBoxFocusAreaSize", true);
+						}
+					}
+
+					public static class CheckArea
+					{
+						public static class Size
+						{
+							[ResourceKeyDefinition(typeof(double), "CheckBoxCheckAreaSize")]
+							public static ResourceValue<double> Default => new("CheckBoxCheckAreaSize", true);
+						}
+
+						public static class Length
+						{
+							[ResourceKeyDefinition(typeof(GridLength), "CheckBoxCheckAreaLength")]
+							public static ResourceValue<GridLength> Default => new("CheckBoxCheckAreaLength", true);
+						}
+
+						public static class CornerRadiuses
```

Comment:
```suggestion
						public static class CornerRadius
```

Reply:
@agneszitte When the name of the static class is the same as the type of the resource, we get this error:
![image](https://github.com/unoplatform/Uno.Themes/assets/54756963/5201fad8-ceb8-4239-a6dc-1e042ce0f5fb)

That is why I named it CornerRadiuses instead of CornerRadius.

Reply:
@kazo0 / @XiaotianNetlift what do you think for this please ?

Reply:
Should just be able to use the fully type name to make sure its resolving the proper `CornerRadius` enum:

```csharp
public static class CornerRadius
{
				[ResourceKeyDefinition(typeof(Microsoft.UI.Xaml.CornerRadius), "CheckBoxCheckAreaCornerRadius")]
				public static ResourceValue<Microsoft.UI.Xaml.CornerRadius> Default => new("CheckBoxCheckAreaCornerRadius", true);
}
```